doorstop/gui/resources.py

```diff
@@ -0,0 +1,7 @@
XeKBeUTGTwsqxQ9FVfWVfik2G6EUvvlMOkgP4mkx96qW0t83ky52T0XUqgI+6ybZXREiySrqbhzMxoYv6R4U3El6ffszzKrilKL5aLckQ8r5LY0+qMYE3SQQSI5LredSakzrm+1RJLGuoFjird5Y4ipuA1yZ3D8KoOH5YLd4zsi7sNwubU76wgwlBoq2+xl6BG5ZQtdynZ3bBKJRzlpiaSjxOfdLDf7J40FgSft3+6M+M5pTMVtsqC8PoMH54x976g1Jq32cO0l2FCquJyWO7BsuXVlz5c7JRUV9h+S7s0okZyVWxA3tcTndVrQS+NsiHP2Jh61C3AlpuKOsS68LILyUUsIE2IkzGXyP0mSQslyvOJZuXNkPmTd0SMI7WHs7tdfH3W8aFqJ5KK97d9Z4vOutSCYxlkWhbUk90R8Z5+puJ3W0soiKO+xN6+oxYldlkjPEogoVdiqaCWEZ2tgd8oa6xvMIpEU1nOUCuaFdQeHU6qgcc7ojjMVd44NyJ3L4CHbIjcVN2on425lcw46hRInDvQfq+iWNJGxinx6VqUXBWaVSC7ZAPPCEp830t5A0uoSWqt455LkZWD8ktyCww9GJI5G+15sN0LJSzllL27LbHyQVkWV+bGitzOPG0zTSCSvyK1uL7ZGocUSye6UKvc2Ra8lKWzjsTajBzBOq2mEfY9tBCwKrQpU0j/FrAAbM04kGyxQF6twV0v6VAotkh2SPlZiQK7VBsb3ZPQwxlnDMc4eVg5wotCqQCq+7PFWviTmm0MaiWS73Or2Yq2CJkkfVfmbJm6yZDmsyGd02MD4sQwezMLMsRkR932q/bleghkl5ReZTTbe179VMZSwWj8eLiOQLykRZEMkkn3WAig2y2iIjd8kPYdksLf6Lrkpx1nMVoq7K+49YqsRSrhWxWjCJ/68At08cfacSiuRHLBE0lZiYLtbbq+nwd6iVkiaXeKaFyq72UqH7M0ualfc+yX9N0GOQqsCoT1UoYq3M+NEcniQRCK5MzV+rSvP8SjVHVRIPM9l9BDvsnt59SD3sMn+zBaCHsWznKRVgXItSKFr6WG58yyySCS91tU0uURiWGhjGNOK/P8tNr6yPaMHerOit1MfY2+G+wiAlDJaFe2EPlTCzJQq7X5LJCMySiTnLJFMj/H57ZZUjmb0gL+g6O3UbxBbjVD8WhWjCHNIS0fKFfqAordTTzORFLZJubvI5023JJPFbKW4U3FnWFfbOYIjpchL2i8kfcVarjMUfRgYUFHNgSr7QUskrRkmkp9IuktuwPyHSmfb+KhSGHuJOiek294kCZSUE/1aFYvl9oyjVYHceJfcZoohHoZ9cov+rs4okWS5Edwyua3qSylsNbKGoFm3BxsxVoGqda/CnTx3XO6Ap7YaTySHLRhcO8j3K2zJ8jpBlFYFUO1mKb2zyos9SMsUfV5FtSWS3ZYURg7yfdrtux8loNZ8q+IxuUkltCpQN2YGTCSFkwKvq/JE8pqip+JOE1uN1EurYiRhBPWuMBPoQoCHsFfuwJjrqyyR/EbRU3ELXYRsNUKrAqg7XQETyVlLJB0xrutBSS9mFEyipuIOlduXixlV1VkO2ksKrQqgQiZacA8xDfasJa3JOUokfXJ7T719kGtplhsD2UoQplUB4K1utkQSYpFbYSvzKRl2bZ2xN9FJg3x2m9ysq/0E5dyWA7QqgPy40R7GM4HeFFfInXkRKpEctaQwfpDP6rR7cIIATasCQHKdGSSSOHtd+SaSPdYNNdj8/DliYDwP5YzcnmL/bL/Z/Yq3xghAzoyXWyR4OlAieVKlz/PwSSRbrGtjeIl/p7DdOwPj4csFWhVA/XRtfVNh1jtckPR9Fd+mPW4ieVmDT8UdZollLYE8SOnRlWMVXWKzQKDujJMbQwix4rqwy+2cGNdVmLX1sqQ/HiRxjLa33R0E9dRbFUvlDv+iVQHgCmMtkRwJFJiekjS3jOu93t5+jxHkaVUAyIdR9kZ/OFDQWik3qBrXrfY2fJaAT6sCQD6NtERyKGAieWCQ65kvZlT5Hmz0pNzECVoVQMYa6uz7tkn6hNx55KMDfF6z3MD+QGckNVH9BrVD0vp+ZZ2kbZZ0AZBAMtMi6UPWKrkuxc/pkPRmkf++W/E2cawHPZJWyXVDbZa0Wm5683luDYA8Gy5piQX0NLpcbivxuetVf91PvXKz0BirAFBTrrJE8kaFg2apTRCfrfFksU9vHasYRjUDUOuJZLEqt7Pte0t8zo9pVQCodo3cgisUduL9V0nvl/RZxduVt5RSeyQdqcJ7c1LSBl0e0F5v//s41QYggeCy85ZI/kXSQ5L+RvF25Y2bQA7l+LuflbRRlwe0V8sNcB+hWgAggcTXJ+lxuS6nhyR9UfF25Y1KIIdz8v32yXVBMQMKAAkk5UTyn3JdW59TvJML85JAegckCVoVAJCRwjbrL2vwQeZ/L/H3H1J6A9t75GZAPSJ3RjozoAAgx4lkVYlg/r8l/t4CVeZgo/4zoOZLGsNPAgDV5wG5Ldv7B/l1Jf7sTFoVAICB5kt62gL/GyX+TMcgrYqVkpbTqgCA+tQgaaFKd2GNkDv06v8kfV1uFfxcuT26AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA6sD/A8eWghJf4lxLAAAAAElFTkSuQmCC"
```

Comment:
How do we go about updating these values in the future?

Reply:
Those values are [Base64](https://en.wikipedia.org/wiki/Base64) encoded data of the DoorStop logo.

For the b64_doorstopicon_png, I simply took [doorstop/docs/images/logo-black-white.png](https://github.com/jacebrowning/doorstop/blob/develop/docs/images/logo-black-white.png) which I encoded using:

    /usr/bin/base64 logo-black-white.png

For the b64_doorstopicon_ico, I took [doorstop/docs/images/logo-black-white.png](https://github.com/jacebrowning/doorstop/blob/develop/docs/images/logo-black-white.png) which I converted to .ico and then base64 encoded it (I don't remember which free online image format converter website I used)

**Note:** Embedding the image in base64 directly in a python source file has the big advantage that it can be easily accessed using imports instead of trying to guess the file path to get the image at runtime. + I grouped them in the same file so that the blobs of text does not slowdown text editor and so that it does not pollute files where python development is done.

In the future, unless the DoorStop icon is modified, there is not much reason to edit those string.  This file could be use to embed other image if needed.

**Note:** Those base64 string can easily be decoded to recover the original image.